### PR TITLE
Sort output

### PR DESCRIPTION
**Pull request by avh4**
This makes the output file have a deterministic sort order.  This will mean that the hash of the output file can then be used to determine whether or not the file has been meaningfully changed.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -156,8 +156,17 @@ module.exports = {
   // TODO check config
   buildElmAssets: (config, callback) => {
     validateConfig(config);
-    collectAssets(config, (err, assets) =>
-      writeElmFile(config, assets, callback));
+    collectAssets(config, (err, assets) => {
+      var sortedAssets =
```
Comment (ento) [inline index.js]: 💭 I wonder if `collectAssets` can/should take on the responsibility of sorting assets. I like that `buildElmAssets` was a sequence of calls to exported functions.

**Comment by avh4** [comment]
(does someone on ops & monitoring have time to take this over?  I don't know if I will get back to it right away)

**Comment by ento** [comment]
todo

- [x] wait for Jenkins to get past the run-jetpack.sh stage in a fresh workspace (build 10)
- [ ] wait for Jenkins to get past the run-jetpack.sh stage in a non-fresh workspace
- [ ] rebase out the TEMP commit
- [ ] merge the PR in build-elm-assets and publish a release
- [ ] merge this PR

**Comment by stoeffel** [comment]
published with v1.2.1

**Pull request merged**